src/main/java/teammates/ui/webapi/output/StudentsData.java

```diff
@@ -10,7 +10,11 @@
  */
 public class StudentsData extends ApiOutput {
 
-    private final List<StudentData> students;
+    private List<StudentData> students;
+
+    public StudentsData() {
+        // Use this constructor if StudentData is to be sent with the key
+    }
```

Comment:
There's an issue of type erasure though, cos the other constructor takes in `List<StudentAttributes>`. I could make these constructor instantiate an empty StudentsData array though, just in case

Reply:
Oh right I didn't consider that, let's just stick with this but maybe add a comment to explain/warn future devs? An empty list doesn't really solve the problem either 😞